src/Base/Core/Interval.hs

```diff
@@ -149,35 +147,35 @@ instance Invertible Interval where
       newI = normalizeIntervalSize $ 9 - normalizeIntervalSize i
       newQual =
         case iQual of
-          Major          -> Minor
-          Minor          -> Major
-          Perfect        -> Perfect
-          (Augmented x)  -> Diminished x
-          (Diminished x) -> Augmented x
+          Major          -> minor
+          Minor          -> major
+          Perfect        -> perfect
+          (Augmented x)  -> fromJust $ diminished x
+          (Diminished x) -> fromJust $ augmented x
```

Comment:
Check it out! We can pattern match using the old constructor names, but we can't actually _use_ those constructors to make new values of `IQuality`! :) Exactly what we were looking for.